ONSdigital/ras-rm-csv-worker
# handle error when row is failed to be processed correctly

# What and why?
This stops `csv-worker` from panicking and crashing if the sample file that was uploaded was able to get through the `sample-file-uploader` but the row is still in the wrong format.
# How to test?
- Deploy this PR and the `sample-service` PR on the trello card
- Download the broken sample file linked in the trello card
- Create a collection exercise and upload the sample file
- An error message should replace the loading text telling the user there is an issue with the file formatting

Note: for this to work the concourse pubsub job needs to be ran again while the terraform branch is set to `https://github.com/ONSdigital/ras-rm-terraform/pull/221`
# Trello
https://trello.com/c/bImN6iK4/1547-problems-with-loading-a-sample
